app/client/reducers/transferAccount/actions.ts

```diff
@@ -0,0 +1,107 @@
+import {
+  TransferAccountActionTypes,
+  LoadTransferAccountActionTypes,
+  EditTransferAccountActionTypes,
+  SetTransferAccountActionTypes,
+  TransfersByUserId
+} from "./types";
+import { CreditTransfer } from "../creditTransfer/types";
+import { createAction, ActionsUnion } from "../../reduxUtils";
+
+export const setSelected = (selected: string[]) =>
+  createAction(SetTransferAccountActionTypes.SET_SELECTED, selected);
+
+export const resetSelected = () =>
+  createAction(SetTransferAccountActionTypes.RESET_SELECTED);
+
+export const SetTransferAccountAction = {
+  setSelected,
+  resetSelected
+};
+
+export type SetTransferAccountAction = ActionsUnion<
+  typeof SetTransferAccountAction
+>;
+
+export const deepUpdateTransferAccounts = (
+  transfer_accounts: TransfersByUserId
+) =>
+  createAction(
+    TransferAccountActionTypes.DEEP_UPDATE_TRANSFER_ACCOUNTS,
+    transfer_accounts
+  );
+
+export const updateTransferAccounts = (transfer_accounts: TransfersByUserId) =>
+  createAction(
+    TransferAccountActionTypes.UPDATE_TRANSFER_ACCOUNTS,
+    transfer_accounts
+  );
+
+export const updateTransferAccountsCreditTransfers = (
+  credit_transfer_list: CreditTransfer[]
+) =>
+  createAction(
+    TransferAccountActionTypes.UPDATE_TRANSFER_ACCOUNTS_CREDIT_TRANSFERS,
+    credit_transfer_list
+  );
+
+export const TransferAccountAction = {
+  deepUpdateTransferAccounts,
+  updateTransferAccounts,
+  updateTransferAccountsCreditTransfers
+};
+
+export type TransferAccountAction = ActionsUnion<typeof TransferAccountAction>;
+
+export const loadTransferAccountsRequest = (payload: any) =>
+  createAction(
+    LoadTransferAccountActionTypes.LOAD_TRANSFER_ACCOUNTS_REQUEST,
+    payload
+  );
+
+export const loadTransferAccountsSuccess = (lastQueried: Date) =>
+  createAction(
+    LoadTransferAccountActionTypes.LOAD_TRANSFER_ACCOUNTS_SUCCESS,
+    lastQueried
+  );
```

Comment:
I think these payloads should be more explicit to take full advantage of Typescripts benefits. Console log the payloads and server response to see the data structure. See CreditTransfer actions for examples:

https://github.com/teamsempo/SempoBlockchain/blob/bb62b22656b8423e537a866464bedcd488b5f3c2/app/client/reducers/creditTransfer/actions.ts#L34-L47